plugin/src/_rules/position.js

```diff
@@ -0,0 +1,24 @@
+import { handler as h, insetMap } from '#utils';
+
+function handleInsetValue(v, { theme }) {
+    return theme.spacing?.[v] ?? h.fraction.auto(v); // TODO: warn if no value
+}
+function handleInsetValues([, d, v], ctx) {
+    const r = handleInsetValue(v, ctx);
+    if (r != null && d in insetMap)
+        return insetMap[d].map(i => [i.slice(1), r]);
+}
+export const insets = [
+    [/^inset-(.+)$/, ([, v], ctx) => ({ inset: handleInsetValue(v, ctx) }),
+        {
+            autocomplete: [
+                '(inset-<directions>-$spacing',
+                'inset-(block|inline)-$spacing',
+                'inset-(bs|be|is|ie)-$spacing',
```

Comment:
autocomplete needs cleaned up?

Reply:
Most def, will fix

Reply:
[be1322a](https://github.com/warp-ds/engine/pull/3/commits/be1322a797a93133bf77f44fb7343d2f83452d9a)

This ok or am I missing something, brain is tired, please halp

Reply:
yep that's good!